pegasus/sites.v3/code.org/styles_min/040-page.css

```diff
@@ -294,6 +294,14 @@ input[type="submit"] {
       align-items: center;
       gap: 0.75rem;
     }
+    &.has-external-link {
+      display: inline-flex;
```

Comment:
Let's put a gap here and remove line 302:
```suggestion
      display: inline-flex;
      gap: 0.25rem;
```

Reply:
And then update [this line](https://github.com/code-dot-org/code-dot-org/blob/82cc3c86707023974c95230e821d6e451c48036e/pegasus/sites.v3/code.org/styles_min/040-page.css#L299) to `gap: 0.25rem` per our IRL pixel space discussion.

Reply:
Updated!